examples/DISCO_IOT_SensorDemo/DISCO_IOT_SensorDemo.ino

```diff
@@ -0,0 +1,141 @@
+ For testing the sketch, you can download on the playstore the "BlueNRG"
+ application provided by STMICROELECTRONICS.
+ Launch the application and enable Bluetooth on your smartphone. Connect it to
+ the BLueNRG device. You will see all the services, you can click on each one.
+
+ You can choose the bluetooth MAC address of the device by configuring SERVER_BDADDR.
+
+ Accelerometer values are updated on user action (press user button).
+ Environnemental values (Temperature, humidity and pressure) are updated each seconds.
+ Each minute a notification is sent to the user and seconds can be read.
+
+ */
+
+#include <SPI.h>
+#include <SPBTLE_RF.h>
+#include <sensor_service.h>
+
+/* Configure SPI3
+  MOSI: PC12
+  MISO: PC11
+  SCLK: PC10
+  */
+SPIClass SPI_3(44, 43, 42);
+
+// Configure BTLE pins
+SPBTLERFClass BTLE(&SPI_3, 50, 57, 31, LED4);
+
+const char *name = "BlueNRG";
+uint8_t SERVER_BDADDR[] = {0x12, 0x34, 0x00, 0xE1, 0x80, 0x03};
+
+AxesRaw_t axes_data;
+
+//Data are set with default values but can be replace with data from sensors.
+int16_t temperature = 205;  //20,5°
+int32_t pressure = 102400;  //102400 Pa
+uint16_t humidity = 530;    //53.0%
+
+uint32_t previousSecond = 0;
+
+void setup() {
+  int ret;
+
+  Serial.begin(9600);
+
+  if(BTLE.begin() == SPBTLERF_ERROR)
+  {
+    Serial.println("Bluetooth module configuration error!");
+    while(1);
+  }
+
+  if(SensorService.begin(name, SERVER_BDADDR))
+  {
+    Serial.println("Sensor service configuration error!");
+    while(1);
+  }
+
+  /* Configure the User Button in GPIO Mode */
+  pinMode(USER_BTN, INPUT);
+
+  ret = SensorService.Add_Acc_Service();
+
+  if(ret == BLE_STATUS_SUCCESS)
+    Serial.println("Acc service added successfully.");
+  else
+    Serial.println("Error while adding Acc service.");
+
+  ret = SensorService.Add_Environmental_Sensor_Service();
+
+  if(ret == BLE_STATUS_SUCCESS)
+    Serial.println("Environmental Sensor service added successfully.");
+  else
+    Serial.println("Error while adding Environmental Sensor service.");
+
+  /* Instantiate Timer Service with two characteristics:
+   * - seconds characteristic (Readable only)
+   * - minutes characteristics (Readable and Notifiable )
+   */
+  ret = SensorService.Add_Time_Service();
+
+  if(ret == BLE_STATUS_SUCCESS)
+    Serial.println("Time service added successfully.");
+  else
+    Serial.println("Error while adding Time service.");
+}
+
+void loop() {
+  BTLE.update();
+
+  if(SensorService.isConnected() == TRUE)
+  {
+    //Update accelerometer values
+    User_Process(&axes_data);
+
+    //Update time
+    SensorService.Update_Time_Characteristics();
+
+    if((millis() - previousSecond) >= 1000)
+    {
+      //Update environnemental data
+      previousSecond = millis();
+      SensorService.Temp_Update(temperature);
+      SensorService.Press_Update(pressure);
+      SensorService.Humidity_Update(humidity);
```

Comment:
I would prefer to send a random value here in order to see something updating in the app.
For example : 
      SensorService.Temp_Update(random(-100,400));
      SensorService.Press_Update(random(95000,105000));
      SensorService.Humidity_Update(random(0,1000));